src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/LoggingTranslatorPass.php

```diff
@@ -28,6 +28,9 @@ public function process(ContainerBuilder $container)
         if (!$container->hasAlias('logger') || !$container->hasAlias('translator')) {
             return;
         }
+        if ($container->has('profiler')) {
+            $container->hasParameter('translator.logging') && $container->setParameter('translator.logging', false);
```

Comment:
This `&&` is like an "if" - that's uncommon syntax in SF code base, isn't it?
the "hasParameter" check should be moved in the "if" instead.

Reply:
it's not common in SF you are right.